api/zulip/__init__.py

```diff
@@ -134,17 +134,33 @@ def generate_option_group(parser, prefix=''):
                           CA certificates. This will be used to
                           verify the server's identity. All
                           certificates should be PEM encoded.''')
+    group.add_option('--client-cert',
+                     action='store',
+                     dest='client_cert',
+                     help='''Specify a file containing a client
+                          certificate (not needed for most deployments).''')
+    group.add_option('--client-cert-key',
+                     action='store',
+                     dest='client_cert_key',
+                     help='''Specify a file containing the client
+                          certificate's key (if it is in a separate
+                          file).''')
     return group
 
 def init_from_options(options, client=None):
     if options.zulip_client is not None:
         client = options.zulip_client
     elif client is None:
         client = _default_client()
+    if options.client_cert_key:
+        client_cert = (options.client_cert, options.client_cert_key)
```

Comment:
Oh, I see, python-requests expects the weird maybe-tuple API.  It still might be cleaner to convert the two variables to just one just before we call into python-requests, so the weirdness is contained to just that block.